snaploader/src/main/java/electrostatic4j/snaploader/platform/util/NativeVariant.java

```diff
@@ -250,6 +260,46 @@ public static boolean isAMD() {
         public static boolean isARM() {
             return OS_ARCH.getProperty().contains("arm") || OS_ARCH.getProperty().contains("aarch");
         }
+
+        /**
+         * Tests whether the named ISA extensions are all present.
```

Comment:
Can we have more documentation about those CPU extensions and their uses? I know this might be out of context to some sense, but I visualize it as a necessity, and I am very okay if you pick up use cases from Jolt-Jni.

So, it could be:
1) "fma" (bit 12 in the ECX register) could be useful for ....
2) "sse4.1" (bit 19 in ECX register) could be useful for .....
And so on. Is that good for you?

Reply:
I may help you to document these. We also needn't to have them all strictly documented before merging the PR. So, it's really an open timeline request, rather than being very essential to this PR.

Reply:
Just leave it as a last thing to do. So, we won't get distracted thus far.